gravitee-apim-console-webui/src/portal/components/banner-radio-button/banner-radio-button.component.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { MatRadioButton } from '@angular/material/radio';
+
+@Component({
+  selector: 'banner-radio-button',
+  standalone: true,
+  imports: [MatRadioButton],
+  templateUrl: './banner-radio-button.component.html',
+  styleUrl: './banner-radio-button.component.scss',
+})
+export class BannerRadioButtonComponent {
+  @Input()
```

Comment:
Juste info :  you can now use @Input() param to make it as required. to use safety `!`